contracts/BatchReveal.sol

```diff
@@ -350,4 +460,16 @@ abstract contract BatchReveal is
         hasBeenForceRevealed = true;
         emit Reveal(batchNumber, batchToSeed[batchNumber]);
     }
+
+    /// @dev Determines if batch reveal is needed.
+    /// We assume batch reveal is needed if the configuration has
+    /// not been initialized or if revealBatchSize is not 0.
+    function isBatchReveal() internal view returns (bool) {
+        return !isInitialized() || revealBatchSize != 0;
+    }
```

Comment:
I think we could simplify and do:

```suggestion
    function isBatchRevealEnabled() internal view returns (bool) {
        return revealBatchSize != 0;
    }
```

and if we want to verify isInitialized and isBatchRevealedEnabled, we will call both? It seems a bit more readable